target/linux/generic/pending-4.9/191-MIPS-Use-randomize_page-in-arch_randomize_brk.patch

```diff
@@ -0,0 +1,74 @@
+From 439551075b16726ea775ea0d62cae6817626f8aa Mon Sep 17 00:00:00 2001
+From: Julien Dusser <julien.dusser@free.fr>
```

Comment:
Are you planning to send this patch for Linux mainline integration?
I do not have enough MIPS low level knowledge and would only apply this if the MIPS experts say this is ok. 
The patch you backported looks ok to me.